### PR TITLE
Fix bad error and add new sbt config

### PR DESCRIPTION
**Pull request by aweng98**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/monitor/SubmissionDetailsPage.scala [hunk 1]
```diff
@@ -85,7 +87,7 @@ class SubmissionDetailsPage(namespace: String, name: String, var submissionId: S
         if (isSubmissionDone) Some(true) else None
       }
     }) match {
-      case None => throw new Exception(s"Workflow Submission $submissionId failed")
+      case None => throw new TimeoutException(s"Timed out ($timeOut) waiting for submission $submissionId to complete")
```
Comment (jmthibault79) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/monitor/SubmissionDetailsPage.scala:90]: yay thank you

File: automation/project/Settings.scala [hunk 2]
```diff
@@ -27,7 +27,7 @@ object Settings {
   )
 
   val testSettings = List(
-    testOptions in Test += Tests.Argument("-oF", "-u", "test-reports")
+    testOptions in Test += Tests.Argument("-oFDG", "-u", "test-reports", "-fW", "test-reports/summary.log")
```
Comment (jmthibault79) [inline automation/project/Settings.scala]: Add a comment explaining what all these arguments do

Comment (aweng98) [inline automation/project/Settings.scala]: `D` - Show test duration after test name.
`G` - Show all failed tests and their full stacktrace at the end of log file so I don't have to scroll to find stacktrace.
`-fW` - Write logs in summary.log without colored text.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala [hunk 3]
```diff
@@ -18,6 +18,5 @@ trait Clickable extends LazyLogging { this: Component =>
     }
     scrollToVisible()
     click on query
-    logger.debug(s"clicked on query:  '$query'") // debugging - remove this once done with all flaky tests
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala:21]: not needed for now.

**Review by jmthibault79 — APPROVED** [review]

**Review by MattPutnam — APPROVED** [review]

**Comment by aweng98** [comment]
jenkin retest

**Comment by aweng98** [comment]
jenkins retest

**Pull request merged**